playbooks/openstack-cloud-controller-manager-unit-test/run.yaml

```diff
@@ -0,0 +1,23 @@
+- hosts: all
+  become: yes
+  roles:
+    - clone-devstack-gate-to-workspace
+    - create-devstack-local-conf
+    - install-devstack
+
+  tasks:
+    - shell:
+        cmd: |
+          set -e
+          set -x
+          source /opt/stack/new/devstack/openrc admin admin
+          openstack server create --flavor 1 --image cirros-0.3.5-x86_64-disk --network private test
```

Comment:
I prefer to create and clean up server in test case, I have log an issue in https://github.com/dims/openstack-cloud-controller-manager/issues/73 , let us waiting dims opinion.